configs/base/index.js

```diff
@@ -137,6 +137,8 @@ module.exports = {
       "error",
       { object: "document", property: "querySelector" },
       { object: "document", property: "querySelectorAll" },
+      { object: "window", property: "outerWidth" },
+      { object: "window", property: "outerHeight" },
```

Comment:
relates to [above comment](https://github.com/mediamonks/eslint-config/pull/28/files#r797672604)
```suggestion
      { object: "window", property: "outerHeight", message: "Did you mean to use `innerHeight` ?" },
```